docs/rules/symbol-description.md

```diff
@@ -0,0 +1,94 @@
+# Require/Disallow Symbol Description (symbol-description)
+
+The `Symbol` constructor may have optional description:
```

Comment:
Good point. Then `no-new-symbol` should be fixed as well - I based this one on that.

> The `Symbol` constructor is not intended to be used with the `new` operator

I'll fix it as well.